src/neurobio/sleep_consolidation.py

```
@@ -0,0 +1,278 @@
+                                noisy_input
+                            )
+                else:
+                    continue
+
+                # Replay experience (forward pass only, no backprop)
+                with torch.no_grad():
+                    try:
+                        _ = self.model(noisy_input.unsqueeze(0))
+                    except Exception:
+                        # Skip if error (e.g., sequence too long)
+                        continue
```

Ruff (0.14.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)